building_energy_standards_data/database_files/hvac_minimum_requirement_heat_rejection_90_1_prm.json

```diff
@@ -0,0 +1,112 @@
+[
+    {
+        "template": "90.1-PRM-2004",
+        "equipment_type": "Open Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 38.2,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2004 Section G3.1.3.11 and Table 6.8.1G"
+    },
+    {
+        "template": "90.1-PRM-2004",
+        "equipment_type": "Closed Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 38.2,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2004 Section G3.1.3.11 and Table 6.8.1G"
+    },
+    {
+        "template": "90.1-PRM-2007",
+        "equipment_type": "Open Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 38.2,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2007 Section G3.1.3.11 and Table 6.8.1G"
+    },
+    {
+        "template": "90.1-PRM-2007",
+        "equipment_type": "Closed Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 38.2,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2007 Section G3.1.3.11 and Table 6.8.1G"
+    },
+    {
+        "template": "90.1-PRM-2010",
+        "equipment_type": "Open Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 38.2,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2010 Section G3.1.3.11 and Table 6.8.1G"
+    },
+    {
+        "template": "90.1-PRM-2010",
+        "equipment_type": "Closed Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 14,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2010 Section G3.1.3.11 and Table 6.8.1G"
```

Comment:
I understand why you provided this entry as well as the `Open Cooling Tower` one. According to the PRM RM (see [here](https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-25130.pdf) on page 235), the Appendix G baseline model should be an `Open Cooling Tower` so I think that it is safe to remove these entries for 90.1-2004 through 90.1-2013 to avoid any confusion from Appendix G implementers or users.

Reply:
Thanks! Indeed the language is different in the 90.1-2004 through 90.1-2010 Appendix G as they state "The heat rejection device shall be an axial fan cooling tower with twospeed fans." without specifying whether it is open or closed.

Will push a new commit.